nireports/tests/test_reportlets.py

```diff
@@ -486,3 +487,41 @@ def test_plot_raincloud(orient, density, tmp_path):
         nans_value=nans_value,
         output_file=output_file,
     )
+
+
+def test_plot_fd(request, tmp_path, outdir):
+    rng = request.node.rng
+
+    fd_file = os.path.join(tmp_path, "fd.txt")
+    fd_radius = 50
+    mean_fd_dist = rng.random(100)
+
+    # Simulate FD file content
+    with open(fd_file, "w") as f:
+        for _ in range(100):
+            f.write(" ".join(map(str, rng.random(6))) + "\n")
+
+    fig = plot_fd(fd_file, fd_radius, mean_fd_dist=mean_fd_dist)
+    assert fig is not None
+    if outdir is not None:
+        fig.savefig(outdir / "test_plot_fd.svg")
+    plt.close(fig)
+
+
+def test_plot_dist(request, tmp_path, outdir):
+    rng = request.node.rng
+    distribution = rng.random(100)
+    xlabel = "Distribution"
+    xlabel2 = "Another Distribution"
+
+    fig = plot_dist(
+        get("MNI152NLin2009aAsym", resolution=1, suffix="T1w"),
+        get("MNI152NLin2009aAsym", resolution=1, desc="brain", suffix="mask"),
+        xlabel,
+        distribution=distribution,
+        xlabel2=xlabel2,
+    )
+    assert fig is not None
+    if outdir is not None:
+        fig.savefig(tmp_path / "test_plot_dist.svg")
```

Comment:
Sorry, I may have forgotten to also update the path:
```suggestion
        fig.savefig(outdir / "test_plot_dist.svg")
```